content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst

```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
-:guilabel:`Force Removal Strategy` to :abbr:`FEFO (First Expired, First Out)`.
+Navigate to the product category, `Wheats`, by going to :menuselection:`Inventory --> Configuration
+--> Product Categories` and selecting the desired category. On the product category form, set the
+:guilabel:`Force Removal Strategy` to :guilabel:`Least Packages`.
 
-.. image:: removal/fefo.png
+.. image:: removal/set-least-packages.png
    :align: center
-   :alt: FEFO forced removal strategy set on the product category.
-
-Next, go to the :menuselection:`Sales app` and click :guilabel:`Create` to create a sales order.
-Next, select a :guilabel:`Customer` from the drop-down menu. Then click :guilabel:`Add a product` in
-the :guilabel:`Order Lines` tab. Select a product (for this example, the :guilabel:`Hand Cream`)
-from the drop-down menu, or type in the name of the product in the field. Enter a quantity (in this
-example, `25.00`) in the :guilabel:`Quantity` field, then click :guilabel:`Save`, then click
-:guilabel:`Confirm`.
-
-Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
-the lot numbers expiring first will be reserved thanks to the :abbr:`FEFO (First Expired, First
-Out)` strategy. All twenty tubes of hand cream from lot `0000001` and five from lot `0000003` will
-be selected to be sent to the customer, detailed in the :guilabel:`Detailed Operations` tab in the
-sales order.
-
-.. image:: removal/pick-hand-cream.png
+   :alt: Set least packages removal strategy on the product category.
+
+Create a :ref:`delivery order <inventory/delivery/one-step>` for eighty kilograms of flour, by
+either going to the :menuselection:`Sales app` and create a new quotation, or from the delivery
+orders dashboard in :menuselection:`Inventory --> Operations --> Deliveries`.
+
+On the delivery order, the :guilabel:`Quantity` field displays the amount automatically picked
+according to the removal strategy. For more details about *where* the units were picked, select the
+:guilabel:`⦙≣ (bulleted list)` icon on the far right. Doing so opens the :guilabel:`Stock move`
+pop-up window that displays how the reserved items were picked according to the removal strategy.
+
+In the :guilabel:`Stock move` window, the :guilabel:`Pick from` field details where the quantities
+to fulfill the :guilabel:`Demand` are picked from. Since the order demanded eighty kilograms, which
+exceeds the quantity in the opened package of `54 kg`, an unopened package of `100 kg` is selected.
+
+.. image:: removal/least-package.png
    :align: center
-   :alt: Hand cream lot numbers selected for the sales order.
+   :alt: Show which package was picked in the *Pick From* field.
+
+Doing so allows the warehouse employees to consolidate the remaining `20 kg` of flour from the `100
+kg` package with the `54 kg`, instead of picking the `54 kg` package, and picking the remaining `26
+kg` from another sealed package.
```

Comment:
I would remove this. It seems outside the scope of the removal strategy.

Reply:
```suggestion

```